domainservice/src/test/kotlin/ua/marchenko/artauction/auction/controller/nats/GetAuctionByIdNatsControllerTest.kt

```diff
@@ -0,0 +1,59 @@
+package ua.marchenko.artauction.auction.controller.nats
+
+import artwork.random
+import auction.random
```

Comment:
```suggestion
import ua.marchenko.artauction.artwork.random
import ua.marchenko.artauction.auction.random
```

Reply:
Random method is in the testFixtures `package auction`, but `ua.marchenko.artauction.auction` is the main package, so I can't change it (without changing package in testFixtures and imports in my all test)

Reply:
done in new PR